js/app.js

```diff
@@ -95,17 +99,41 @@ newStoreForm.addEventListener('submit', function(event) {
   if(!event.target.store.value && !event.target.newMin.value && !event.target.newMax.value && !event.target.newAvg.value) {
     return alert('Cannot be empty');
   }
+  //check to see if new store name equals an already existing store
   var newStoreName = event.target.store.value;
   var newMinimum = parseFloat(event.target.newMin.value);
   var newMaximum = parseFloat(event.target.newMax.value);
   var newAverage = parseFloat(event.target.newAvg.value);
-  var newStore = new Shops(newStoreName, newMinimum, newMaximum, newAverage);
+  var existingStore = false;
+  for(var i = 0; i < shopObjects.length; i++) {
+    if(newStoreName === shopObjects[i].locationName) {
+      existingStore = shopObjects[i];
+      break;
+    }
+  }
+  // if no existing store is found
+  if (!existingStore) {
+    var newStore = new Shops(newStoreName, newMinimum, newMaximum, newAverage);
+    newStore.generateCookiesForEachHour();
+    shopObjects.push(newStore);
+    //renderTableData(newStore);
+  } else { // if store is found
+    existingStore.minCustPerHour = newMinimum;
+    existingStore.maxCustPerHour = newMaximum;
+    existingStore.avgCookiesPerCust = newAverage;
+    existingStore.generateCookiesForEachHour();
+  }
+
+  document.getElementById('cookies').innerHTML = ''; // clear out table
```

Comment:
@kaylynyuh nice clear before calling the render methods!  This type of thing is super common.